h/form.py

```diff
@@ -15,16 +20,29 @@
 )
 
 
-def init():
-    from deform import Form
-    from deform_jinja2 import jinja2_renderer_factory
-    from deform_jinja2.translator import PyramidTranslator
+# We supply our own Jinja2 renderer factory because deform_jinja2's one doesn't
+# support autoescaping.
+def jinja2_renderer_factory(config):
+    environment = []
+
+    def jinja2_renderer(template_name, **kwargs):
+        if not environment:
+            searchpath = [pkg_resources.resource_filename(*(path.split(':')))
```

Comment:
We should probably be using `pyramid.path.AssetResolver` to do this rather than this hacky version from `deform_jinja2`.

Something like:

```python
from pyramid.path import AssetResolver

resolver = AssetResolver()

...
    searchpath = [resolver.resolve(p).abspath() for p in SEARCH_PATHS]
```